scripts/unexpire_layer_version.py

```diff
@@ -0,0 +1,22 @@
+import boto3
+from boto3.dynamodb.conditions import Key
+
+
+session = boto3.Session(profile_name='KlayersDefault')
+dynamodb = session.resource("dynamodb")
+table = dynamodb.Table("kl.Klayers-defaultp38.db-ver2")
+
+region = "ap-southeast-1"
+package = "requests"
+python_version = "p3.9"
+
+pk = f"lyr#{region}:{package}:{python_version}"
+
+kwargs = {
+    "KeyConditionExpression": Key("pk").eq(pk),
+    "FilterExpression": "attribute_exists(dplySts)",  # don't get latest version
+}
+
+response = table.query(**kwargs)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `query` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `ExclusiveStartKey`, `IsTruncated`, `LastEvaluatedKey`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)